src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -258,52 +429,90 @@ class BaseDropdown extends Component<DropdownProps> {
           >
             {labelText}
           </Label>
-          <ListboxInput
-            {...getConditionalAriaProp('aria-labelledby', [
-              ariaLabelledBy,
-              labelId,
-            ])}
-            disabled={disabled}
-            onChange={onChange}
-            ref={forwardedRef}
-            name={name}
-            value={selectedValue || ''}
-          >
-            <ListboxButton
-              ref={this.buttonRef}
-              {...dropdownButtonProps}
+          <HintText id={hintTextId}>{hintText}</HintText>
+          <HtmlDiv className={classnames(dropdownClassNames.inputWrapper)}>
+            <HtmlSpan
```

Comment:
Could this element be button, since it has role button? Could also then move part of onKeyDown logic to onClick

Reply:
Changed the element to be a button. 

onMouseDown was used as a trick so that focus ring does not disappear for a split second. But now it seems onClick works as well 🤷